flocker/restapi/_logging.py

```diff
@@ -36,5 +36,5 @@
 JSON_REQUEST = ActionType(
     LOG_SYSTEM + u":json_request",
     [JSON],
-    [RESPONSE_CODE, JSON],
+    [RESPONSE_CODE],
     u"A request containing JSON request and response bodies.")
```

Comment:
❓ Is this description still correct, should we be referring to the response body here?